packages/appsync-emulator-serverless/lambdaRunnerPython.js

```diff
@@ -0,0 +1,69 @@
+const log = require('logdown')('appsync-emulator:lambdaRunner');
+
+const parseErrorStack = error =>
+  error.stack
+    .replace(/at /g, '')
+    .split('\n    ')
+    .slice(1);
+
+const sendOutput = output => {
+  process.send({ type: 'success', output }, process.exit);
+};
+const sendErr = err => {
+  let error;
+  if (err instanceof Error) {
+    error = {
+      stackTrace: parseErrorStack(err),
+      errorType: err.constructor.name,
+      errorMessage: err.message,
+    };
+  } else {
+    error = err;
+  }
+  process.send({ type: 'error', error }, process.exit);
+};
```

Comment:
Since our lambda runners have a similar API, perhaps it makes sense to include a `./lambda` directory as well as a `./lambda/util.js` file in order to have functions like `parseErrorStack`, `sendOutput` in a single place

Reply:
:+1: